src/main/java/com/faforever/client/coop/CoopController.java

```diff
@@ -267,7 +267,10 @@ public void onPlayButtonClicked() {
     modService.getFeaturedMod(COOP.getTechnicalName())
         .thenAccept(featuredModBean -> gameService.hostGame(new NewGameInfo(titleTextField.getText(),
             Strings.emptyToNull(passwordTextField.getText()), featuredModBean, getSelectedMission().getMapFolderName(),
-            emptySet())));
+            emptySet(),
+            0, //Rating isn't interesting for coop games
+            3000,
```

Comment:
constants like 0 and 3000 need to go away... should be something like null, null

Reply:
Good point! I'll implement that.